src/main/java/com/mathworks/ci/Utilities.java

```diff
@@ -23,4 +33,47 @@ public static String getCellArrayFrmList(List<String> listOfStr){
         filteredListOfStr.replaceAll(val -> "'" + val.replaceAll("'", "''") + "'");
         return "{" + String.join(",", filteredListOfStr) + "}";
     }
+
+    public static void addMatlabToEnvPathFrmAxis(Computer cmp, TaskListener listener, EnvVars env)
+            throws IOException, InterruptedException {
+        String name = env.get(Message.getValue("Axis.matlab.key"));
+
+        // If no MATLAB axis is set or if 'Use MATLAB version' is selected, return
+        if (name == null || name.isEmpty() || env.get("matlabroot") != null){
+            return;
+        }
+
+        String matlabExecutablePath = getNodeSpecificHome(name, cmp.getNode(), listener, env) + "/bin";
+        env.put("PATH+matlabroot", matlabExecutablePath);
```

Comment:
Is "PATH+matlabroot" the name of the environment variable?

Reply:
Yes, this is by design. In Jenkins this would prepend matlabroot (represents  matlabExecutablePath) to PATH.